js/game.js

```diff
@@ -1,6 +1,3 @@
-var xhr = new XMLHttpRequest();
-var logger = 'http://129.25.8.18:3080/tasks';
```

Comment:
Yeah that looks a little spooky. I don't see why you wouldn't want to collect game metrics. But using config to store environment variables like is [good practice](https://12factor.net/config).